src/components/FieldNote/FieldNote.tsx

```diff
@@ -36,34 +40,33 @@ export interface Props {
  * ```ts
  * import {FieldNote} from "@chanzuckerberg/eds";
  * ```
+ *
+ * Fieldnote component wraps text to describe other components.
  */
 export const FieldNote = ({
   className,
+  disabled,
   id,
-  iconName,
```

Comment:
Should we also remove this prop? From the code, it looks like this is now being passed down into the wrapper `div` via the `other` variable..

Reply:
Yup will remove